drivers/serial/Kconfig.um201xx

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 Bjarki Arge Andreasen
+# SPDX-License-Identifier: Apache-2.0
+
+config SERIAL_UM201XX
+	bool "FTDI I2C to USB serial converter"
```

Comment:
Also a good idea to add the chip name to the description so it is obvious when people are doing menuconfig.

Reply:
Done :)